kyverno-1.12.yaml

```diff
@@ -1,13 +1,15 @@
 package:
-  name: kyverno
+  name: kyverno-1.12
```

Comment:
Could we check if we need to update any images using this package ?

Reply:
Also we might want to copy the advisory of this new package rename too.

Reply:
> Could we check if we need to update any images using this package ?

I thought passing `provides:` would handle that:
```
provides:
       - kyverno=${{package.full-version}}
```

🤔

Reply:
ops! yeah! never mind. But we need to copy the advisory.

Reply:
I may a bit confused here, should I `cp kyverno.advisories.yaml kyverno-1.12.advisories.yaml` (or `mv`?) and rename `package.name` to `kyverno-1.12`?

Reply:
https://github.com/wolfi-dev/advisories/pull/6585